manifests/v/variar/klogg/22.06/variar.klogg.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: variar.klogg
+PackageVersion: 22.06
```

```suggestion
PackageVersion: 22.06.0.1289
```
This must match Control Panel to prevent upgrade loops due to version mismatch.